ui/apps/dev-server-ui/src/app/(dashboard)/stream/Stream.tsx

```diff
@@ -50,7 +50,12 @@ const columns = [
   }),
   columnHelper.accessor('runs', {
     header: () => <span>Function</span>,
-    cell: (props) => <FunctionRunList functionRuns={props.getValue()} />,
+    cell: (props) => {
+      const functionRuns = props.getValue() as FunctionRun[];
+      const validFunctionRuns = functionRuns.filter((run) => run !== null);
```

Comment:
The types say we could be dealing with nullish values so we should probably handle them:

```suggestion
      let validFunctionRuns: FunctionRun[] = []
      for (const run of props.getValue() ?? [])  {
        if (run) {
          validFunctionRuns.push(run);
        }
      }
```